File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusCouponVO.kt
package com.sdkwork.backend

data class PlusCouponVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val name: String? = null,
    val redeemCode: String? = null,
    val pointCost: Int? = null,
    val type: String? = null,
    val description: String? = null,
    val amount: Int? = null,
    val discount: Double? = null,
    val minConsume: Int? = null,
    val startTime: String? = null,
    val endTime: String? = null,
    val total: Int? = null,
    val getLimit: Int? = null,
    val status: String? = null,
    val scopeType: String? = null,
    val scopeValue: String? = null
)

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_coupon_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusCouponVO:
    """优惠券模板VO，定义优惠券属性"""
    created_at: str = None
    updated_at: str = None
    name: str = None
    redeem_code: str = None
    point_cost: int = None
    type: str = None
    description: str = None
    amount: int = None
    discount: float = None
    min_consume: int = None
    start_time: str = None
    end_time: str = None
    total: int = None
    get_limit: int = None
    status: str = None
    scope_type: str = None
    scope_value: str = None
